pytorch_lightning/utilities/enums.py

```diff
@@ -256,3 +256,22 @@ def interactive_compatible_types() -> List["_StrategyType"]:
     def is_interactive_compatible(self) -> bool:
         """Returns whether self is interactive compatible."""
         return self in _StrategyType.interactive_compatible_types()
+
+
+class FaultTolerantTrainingMode(LightningEnum):
+
+    DISABLED = "0"
+    AUTOMATIC = "1"
+    MANUAL = "2"
```

Comment:
Either these should have actual strings or you should not subclass `LightningEnum`

```suggestion
    DISABLED = "disabled"
    AUTOMATIC = "automatic"
    MANUAL = "manual"
```